deployments/shopify.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+# Deploy to Shopify
+#
+# You can either add these here, or configure them in the environment variables
+# section of your project settings. Information about Shopify Theme Config Variables
+# can be found here - https://github.com/hughker/shopify_theme#configuration
+# You can leave off the SHOPIFY_THEME_ID if you'd like, just remember to remove it
+# from the script before attempting to deploy.
+SHOPIFY_API_KEY=""
+SHOPIFY_API_PASSWORD=""
+SHOPIFY_STORE_URL=""
+SHOPIFY_THEME_ID=""
+
+gem install shopify_theme
+theme configure SHOPIFY_API_KEY SHOPIFY_API_PASSWORD SHOPIFY_STORE_URL SHOPIFY_THEME_ID
```

Comment:
The line should probably read
```
theme configure "${SHOPIFY_API_KEY}" "${SHOPIFY_API_PASSWORD}" "${SHOPIFY_STORE_URL}" "${SHOPIFY_THEME_ID}"
```

though quotes and the braces are optional (I'm just used to using them in most scripts)